ui/components/multichain/account-picker/account-picker.js

```diff
@@ -37,74 +37,67 @@ export const AccountPicker = ({
   addressProps = {},
   labelProps = {},
   textProps = {},
+  className = '',
   ...props
 }) => {
   const useBlockie = useSelector(getUseBlockie);
   const shortenedAddress = shortenAddress(toChecksumHexAddress(address));
 
   return (
     <ButtonBase
-      className="multichain-account-picker"
+      className={classnames('multichain-account-picker', className)}
       data-testid="account-menu-icon"
       onClick={onClick}
       backgroundColor={BackgroundColor.transparent}
       borderRadius={BorderRadius.LG}
       ellipsis
       textProps={{
         display: Display.Flex,
-        gap: 2,
         alignItems: AlignItems.center,
+        gap: 1,
         ...textProps,
       }}
       size={showAddress ? ButtonBaseSize.Lg : ButtonBaseSize.Md}
       disabled={disabled}
+      display={Display.Flex}
```

Comment:
May be able to remove this prop as it's already `Display.InlineFlex`  on `ButtonBase`
```suggestion
```